validation_pipeline.ipynb

```diff
@@ -7,7 +7,7 @@
         "colab_type": "text"
```

Comment:
<p>Are 'fraud' and 'predicted_fraud' class probabilities somehow? I am having trouble understanding how that could be the case.</p><p> </p><p><a href="https://scikit-learn.org/stable/modules/generated/sklearn.metrics.PrecisionRecallDisplay.html#sklearn.metrics.PrecisionRecallDisplay.from_predictions" target="_blank">https://scikit-learn.org/stable/modules/generated/sklearn.metrics.PrecisionRecallDisplay.html#sklearn.metrics.PrecisionRecallDisplay.from_predictions</a> <code>y_true array-like of shape (n_samples,) True binary labels. y_pred array-like of shape (n_samples,) Estimated probabilities or output of decision function.   </code> </p><p> </p><p>I would recommend reusing the results of the p-value sweep you did for the accuracy plot and then plotting precision against recall with pyplot&nbsp;</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tnc-br/ddf-isoscapes/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
So if "fraud" is 1/0 labels and "predicted_fraud" are p-values, I can see how this would make sense. The p-value threshold would take the place of where I usually would see the class probability. If so, can you confirm this is the case, and if so, write a short explanation to this point?

Reply:
If not, please still let me know! And would you also mind putting a P-R curve in the PR?

Reply:
It should be there I really don't know why it doesn't show in the diffs:
https://screenshot.googleplex.com/BmhNRAFAvKEUV9T

Per https://screenshot.googleplex.com/8gphW7cydwLeBEB, predicted_p_value when low should be positive (fraud).

Reply:
This was already how it was implemented before making this change.

Reply:
https://screenshot.googleplex.com/B94r4ByyBC5qCET

This is how it was implemented on main before this change. I don't think we invert any values.